tests/tests/bitasset_tests.cpp

```diff
@@ -1407,4 +1407,138 @@ BOOST_AUTO_TEST_CASE(hf_890_test_hf1270)
+    *
+    * Test prohibitions against changing of the backing/collateral asset for a smart asset
+    * if any collateral asset fees are available to be claimed.
+    */
+   BOOST_AUTO_TEST_CASE(change_backing_asset_prohibitions) {
+      try {
+         /**
+          * Initialize
+          */
+         // Initialize for the current time
+         trx.clear();
+         set_expiration(db, trx);
+
+         // Initialize actors
+         ACTORS((smartissuer)(feedproducer)); // Actors for smart asset
+         ACTORS((jill)(izzy)); // Actors for user-issued assets
+         ACTORS((alice)); // Actors who hold balances
+
+         price price(asset(1, asset_id_type(1)), asset(1));
+         uint16_t market_fee_percent = 20 * GRAPHENE_1_PERCENT;
+         create_user_issued_asset("JCOIN", jill, charge_market_fee, price, 2, market_fee_percent);
+         generate_block(); trx.clear(); set_expiration(db, trx);
+         const asset_object jillcoin = get_asset("JCOIN");
+
+         create_user_issued_asset("ICOIN", izzy, charge_market_fee, price, 2, market_fee_percent);
+         generate_block();
+         const asset_object izzycoin = get_asset("ICOIN");
+
+         // Create the smart asset backed by JCOIN
+         const uint16_t smartbit_market_fee_percent = 2 * GRAPHENE_1_PERCENT;
+         create_bitasset("SMARTBIT", smartissuer.id, smartbit_market_fee_percent,
+                         charge_market_fee, 2, jillcoin.id);
+
+         // Obtain asset object after a block is generated to obtain the final object that is commited to the database
+         generate_block(); trx.clear(); set_expiration(db, trx);
+         const asset_object &smartbit = get_asset("SMARTBIT");
+         const asset_bitasset_data_object& smartbit_bitasset_data = (*smartbit.bitasset_data_id)(db);
+         // Confirm that the asset is to be backed by JCOIN
+         BOOST_CHECK(smartbit_bitasset_data.options.short_backing_asset == jillcoin.id);
+
+         // Fund balances of the actors
+         issue_uia(alice, jillcoin.amount(5000 * std::pow(10, jillcoin.precision)));
+         BOOST_REQUIRE_EQUAL(get_balance(alice, jillcoin), 5000 * std::pow(10, jillcoin.precision));
+         BOOST_REQUIRE_EQUAL(get_balance(alice, smartbit), 0);
+
+
+         /**
+          * Claim any amount of collateral asset fees.  This should fail because claiming such fees are prohibited
+          * before the HARDFORK_CORE_BSIP_87_74_COLLATFEE_TIME.
+          */
+         trx.clear();
+         asset_claim_fees_operation claim_op;
+         claim_op.issuer = smartissuer.id;
+         claim_op.extensions.value.claim_from_asset_id = smartbit.id;
+         claim_op.amount_to_claim = jillcoin.amount(5 * std::pow(10, jillcoin.precision));
+         trx.operations.push_back(claim_op);
+         set_expiration(db, trx);
+         sign(trx, smartissuer_private_key);
+         GRAPHENE_REQUIRE_THROW(PUSH_TX(db, trx), fc::exception);
```

Comment:
Need a test case to verify that it's unable to wrap an `asset_claim_fees_operation` with a non-null `extensions.value.claim_from_asset_id` in a `proposal_create_operation`.